tests/jointcalTestBase.py

```diff
@@ -0,0 +1,311 @@
+# See COPYRIGHT file at the top of the source tree.
+
+from __future__ import division, absolute_import, print_function
+
+import numpy as np
+import collections
+import os
+
+from lsst.meas.astrom import LoadAstrometryNetObjectsTask, LoadAstrometryNetObjectsConfig
+from lsst.afw import geom, table
+
+from lsst.jointcal import jointcal
+
+arcsec_per_radian = 206265.
+
+
+class JointcalTestBase(object):
+    """
+    Base class for jointcal tests, to genericize some test running and setup.
+
+    Derive from this first, then from TestCase.
+    """
+
+    def setUp(self):
+        self.doPlot = False  # don't make plots unless specifically requested
+        self.matchRadius = 0.1*geom.arcseconds  # match sources within 0.1" for RMS statistics
+
+    def tearDown(self):
+        if getattr(self, 'jointcalTask', None) is not None:
+            del self.jointcalTask
+        if getattr(self, 'jointcalTask', None) is not None:
```

Comment:
Jointcal currently fails the MemoryTestCase tests for unknown reasons. See DM-6626. I will try to clean up the tearDown though, thanks.